File: backend-csharp/DTOs/User/UserBriefInfoResponse.cs
namespace SousChefBackend.DTOs.User;

public class UserBriefInfoResponse
{
    public long UserId { get; set; }
    public string UserName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public UserProfile? Profile { get; set; }
}

public class UserProfile
{
    public int? Age { get; set; }
    public int? Height { get; set; }
    public int? Weight { get; set; }
}

File: backend-java/src/main/java/com/souschef/dto/user/UserBriefInfoResponse.java
package com.souschef.dto.user;

import lombok.Data;

@Data
public class UserBriefInfoResponse {
    private Long userId;
    private String userName;
    private String email;
    private UserProfile profile;
    
    @Data
    public static class UserProfile {
        private Integer age;
        private Integer height;
        private Integer weight;
    }
}

